articles/introduction-to-neural-networks/index.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/introduction-to-neural-networks/
+title: Introduction to Deep Learning
+description: This article serves as an introduction to
+author: judy-nduati
+date: 2020-10-19T00:00:00-16:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-neural-networks/hero.jpg
+    alt: deep learning image example
+---
+In this article, we will be talking about neural networks. A functional unit of [deep learning](https://www.section.io/engineering-education/introduction-to-deep-learning/) whose motivation is driven by how the brain works. The technologies give the best solutions to object recognition, speech recognition, pattern recognition, and NLP.
+<!--more-->
+### Introduction
+This article will improve your understanding of neural networks. It will also give you an in-depth interpretation of how neural networks operates. This article will explain distinct kids of neural networks and their applications. By the end of this article, you will have attained an extensive understanding of what neural networks are.
```

Comment:
Also - please revisit this paragraph - each sentence is saying the same thing. 🤔 
It seems very repetitive @JudyNduati